src/events/Ready.js

```diff
@@ -13,16 +13,25 @@ class Ready extends KongouEvent {
         this.client.logger.debug(`${this.client.user.username}`, `Ready! Serving ${this.client.guilds.cache.size} guild(s) with ${this.client.users.cache.size} user(s)`);
         if (!this.interval) {
             await this.client.user.setActivity('It\'s the English-born returnee, Kongou! Nice to meet you!');
+            const ActivityType = [
```

Comment:
Is this because you want specific activity type per playing message? if so, just merge it into a single array. 
Example:
```js
const statuses = [{ type: 'PLAYING', message: 'message here' }];
```

Reply:
Umm yeah but it also do work as same activity type per playing message. Or yes we can use a single array .. But i saw there were about 3 different status so i did it like this.. and it works completely fine

Reply:
Ok i did it like the way you want me to do . please take a look at it once.